libs/entitlement/bridge/build.gradle

```diff
@@ -7,7 +7,10 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
+import org.elasticsearch.gradle.internal.precommit.CheckForbiddenApisTask
+
 apply plugin: 'elasticsearch.build'
+apply plugin: 'elasticsearch.mrjar'
 
 configurations {
```

Comment:
I don't think that configuration is used anywhere

Reply:
It's used in artifacts